packages/api/src/external/carequality/document/process-outbound-document-retrieval-resps.ts

```diff
@@ -56,17 +55,21 @@ export async function processOutboundDocumentRetrievalResps({
     }
 
     for (const docRetrievalResp of results) {
-      newDocRefCount += docRetrievalResp.documentReference?.length ?? 0;
+      if (docRetrievalResp.documentReference) {
+        successDocsRetrievedCount += docRetrievalResp.documentReference.length;
+      } else if (docRetrievalResp.operationOutcome?.issue) {
+        issuesWithGateway += docRetrievalResp.operationOutcome.issue.length;
+      }
     }
 
     await setDocQueryProgress({
       patient: { id: patientId, cxId: cxId },
       downloadProgress: {
-        total: newDocRefCount,
+        total: successDocsRetrievedCount + issuesWithGateway,
```

Comment:
This is so we get all the results count coming in (before we were only accounting for successful ones)